package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@webcomponents/template",
   "version": "0.0.1",
-  "description": "Minimal template polyfills needed by webcomponents.js",
+  "description": "HTML Template polyfill",
```

Comment:
Is it possible to use this project as separate HTML Template polyfill without webcomponents.js? If yes, what difference does it make which project uses it?